src/openvic-simulation/misc/Define.hpp

```diff
@@ -1,87 +1,37 @@
 #pragma once
 
-#include "openvic-simulation/pop/Pop.hpp"
-#include "openvic-simulation/types/IdentifierRegistry.hpp"
-#include "openvic-simulation/types/fixed_point/FixedPoint.hpp"
+#include "openvic-simulation/misc/defines/AIDefines.hpp"
```

Comment:
Move defines directly under openvic-simulation?

Reply:
Ah yeah good idea, I was thinking to do that yesterday but forgot